ui/admin/tests/helpers/selectors.js

```diff
@@ -31,3 +31,9 @@ export const ALERT_TOAST_BODY =
   '[data-test-toast-notification] .hds-alert__description';
 export const ALERT_TOAST_DISMISS =
   '[data-test-toast-notification] .hds-dismiss-button';
+
+export const DIALOG_UNSAVED_CHANGES = '.rose-dialog';
```

Comment:
`.rose-dialog` being something quite generalistic, but for the usage of this selector should be fine. Open to add a `data-test-*` attribute if you think is a better solution or open to discuss other solutions.

Reply:
I think this works. The "quite generalistic" selectors like `.rose-dialog` are the whole reason we created these new selectors. When I read the new selectors name, I immediately knew what it was referring to so I am good with it.